testing/tests/test_macros.rs

```diff
@@ -39,91 +34,81 @@ struct DeNamedMap<A, B, C> {
 }
 
 #[derive(Debug, PartialEq, Serialize)]
-enum SerEnum<'a, B: 'a, C: /* Trait + */ 'a, D> where D: /* Trait + */ 'a {
+enum SerEnum<'a, B: 'a, C: 'a, D>
+    where D: 'a,
+{
     Unit,
-    Seq(
-        i8,
+    Seq(i8,
         B,
         &'a C,
-        //C::Type,
-        &'a mut D,
-        //<D as Trait>::Type,
-    ),
+        // C::Type,
+        &'a mut D /* <D as Trait>::Type, */),
     Map {
         a: i8,
         b: B,
         c: &'a C,
-        //d: C::Type,
-        e: &'a mut D,
-        //f: <D as Trait>::Type,
+        // d: C::Type,
+        e: &'a mut D, // f: <D as Trait>::Type,
     },
 
     // Make sure we can support more than one variant.
     _Unit2,
-    _Seq2(
-        i8,
-        B,
-        &'a C,
-        //C::Type,
-        &'a mut D,
-        //<D as Trait>::Type,
-    ),
+    _Seq2(i8,
+          B,
+          &'a C,
+          // C::Type,
+          &'a mut D /* <D as Trait>::Type, */),
     _Map2 {
         a: i8,
         b: B,
         c: &'a C,
-        //d: C::Type,
-        e: &'a mut D,
-        //f: <D as Trait>::Type,
+        // d: C::Type,
+        e: &'a mut D, // f: <D as Trait>::Type,
     },
 }
 
 #[derive(Debug, PartialEq, Serialize, Deserialize)]
-enum DeEnum<B, C: /* Trait */, D> /* where D: Trait */ {
+enum DeEnum<B, C /* : /* Trait */, D> {
```

Comment:
Also filed https://github.com/rust-lang-nursery/rustfmt/issues/1096 related to this line.

I bet all these comments are remnants of an erickt debugging session so we need to clean them up anyway.